twisted/internet/base.py

```diff
@@ -443,6 +443,13 @@ class ReactorBase(object):
         and C{reactor.stop}.  This should be replaced with an explicit state
         machine.
 
+    @type _lastSignal: None|C{int}|C{bool}
```

Comment:
this one ugly code design... a variable which can take 3 different types... can we have it to use at least 2 types.... int + a guard would be better.